flink-connectors/flink-hbase/src/test/java/org/apache/flink/addons/hbase/HBaseConnectorITCase.java

```diff
@@ -320,6 +336,72 @@ public void testTableSink() throws Exception {
 		TestBaseUtils.compareResultAsText(results, expected);
 	}
 
+	@Test
+	public void testTableSourceSinkWithDDL() throws Exception {
+		StreamExecutionEnvironment execEnv = StreamExecutionEnvironment.getExecutionEnvironment();
+		StreamTableEnvironment tEnv = StreamTableEnvironment.create(execEnv, streamSettings);
+
+		DataStream<Row> ds = execEnv.fromCollection(testData1).returns(testTypeInfo1);
+		tEnv.registerDataStream("src", ds);
+
+		// register hbase table
+		String quorum = getZookeeperQuorum();
+		String ddl = "CREATE TABLE hbase (\n" +
+			"    rowkey INT," +
+			"    family1 ROW<col1 INT>,\n" +
+			"    family2 ROW<col1 VARCHAR, col2 BIGINT>,\n" +
```

Comment:
Use `STRING` instead of `VARCHAR` and `TIMESTAMP(3)`/`TIME(3)` instead of `TIMESTAMP`/`TIME`.

Reply:
`STRING` is not allowed currently? #9354

Reply:
I thought this issue has been fixed for 1.9. This is bad :(

Reply:
This means that the DDL in 1.9 is instant legacy. Can we at least ensure the other comment around `TIMESTAMP(3)/TIME(3)`? Thanks.

Reply:
Sure. I will use `TIMESTAMP(3)/TIME(3)`.